source/git-api.js

```diff
@@ -197,7 +197,8 @@ exports.registerApi = (env) => {
   });
 
   app.get(`${exports.pathPrefix}/diff`, ensureAuthenticated, ensurePathExists, (req, res) => {
-    jsonResultOrFailProm(res, gitPromise.diffFile(req.query.path, req.query.file, req.query.sha1));
+    var whiteSpace = req.query.whiteSpace === "true" ? true : false;
```

Comment:
I think that the `whiteSpace` variables are missleading. Because it is in fact `ignoreWhiteSpace` which is the opposite.